jobs/JGLOBAL_MARINE_ANALYSIS_ECEN

```diff
@@ -63,14 +62,10 @@ fi
 ##########################################
 # Remove the Temporary working directory
 ##########################################
-cd "${DATAROOT}" && true
+cd "${DATAROOT/enkf}" && true
```

Comment:
In order to review this, I will need to run and see these to make an informed comment.